e2e/interchaintestv8/types/attestor_binaries.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+type AttestorBinaryPath = string
```

Comment:
This makes me uncomfortable. I think we can contain this information inside the attestor folder. wdyt?